### PR TITLE
Update wesnoth-ala.po

### PR DESCRIPTION
**Pull request by MultiSeptim**
Next run @celerini

### CODE REVIEW

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 1]
```diff
@@ -2127,6 +2136,9 @@ msgid ""
 "life difficult even if we search for amber pebbles in the depths along the "
 "coast."
 msgstr ""
+"Ložiska jantaru, na ostrově naproti ústí řeky, už neovládáme my, ale "
+"ještěráci! A ty nám ztěžují život, i když hledáme jantarové oblázky v "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"ještěráci! A ti nám ztěžují život, i když hledáme jantarové oblázky v "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 2]
```diff
@@ -2156,6 +2173,13 @@ msgid ""
 "outcome; our community is the smallest of all those present along the coasts "
 "of this continent."
 msgstr ""
+"Během nejchladnějších období se někdy na měsíc nebo dva stěhujeme jižněji "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"Během nejchladnějších období se někdy na měsíc nebo dva stěhujeme "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 3]
```diff
@@ -2267,6 +2302,13 @@ msgid ""
 "those we have crossed to reach the Nereids. If he has come here, he must "
 "have a specific purpose, and I don't think that it's to make a holiday."
 msgstr ""
+"Okamžik, já znám ty modré a žluté standarty! To je kmen Gan-Tarkase! Ve "
+"Weldynu je jeho jméno dobře známé! Je jedním z nejmocnějších a "
+"nejautoritativnějších válečních vůdců. A přestože je oficiálně součástí "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"nejautoritativnějších válečných vůdců. A přestože je oficiálně součástí "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 4]
```diff
@@ -2409,6 +2471,11 @@ msgid ""
 "demand that Wesnoth respect the pacts on the fluvial navigation, while they "
 "were already willing to elude them at the first opportunity!"
 msgstr ""
+"Podívej! Připravil několik člunů naložených balvany, které potopili do řeky "
+"a vytvořily tak brod! Plánování a příprava této bitvy muselo trvat měsíce. "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"a vytvořili tak brod! Plánování a příprava této bitvy musely trvat měsíce. "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 5]
```diff
@@ -3033,7 +3112,18 @@ msgid ""
 "of it, did it with enthusiasm, because he had grown fond of the place, and "
 "the large cities of the west seemed now, to him, suffocating."
 msgstr ""
-
+"Glauko a Geobard odjeli do hlavního města podat svému vznešenému (a trochu "
+"roztržitému) panovníkovi zprávu o tom, co se událo. Poté, co se tak stalo, "
+"zjistil Glauko, že byl povýšen na generála, aniž by předtím sloužil jako "
+"důstojník. Když se ho král zeptal, do jaké posádky chce být přidělen, po "
+"krátkém zamyšlení odpověděl, že Amoran mu bude vyhovovat, protože je tam "
+"zase spousta práce a on nerad nechávám v půli cesty.\n"
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"zase spousta práce a on nerad nechává věci nedodělané.\n"
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 6]
```diff
@@ -3066,6 +3167,12 @@ msgid ""
 "tribe was, in the following years, extraordinarily peaceful, for a tribe of "
 "Ogre at least."
 msgstr ""
+"Glubtrubovi a jeho soukmenovcům od staré přepřahací stanice byly uděleny "
+"trvalé koncese na území, na kterém žili. Pouze Nereidy dostaly povolení "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"trvalé koncese na území, na kterém žili. Pouze nereidy dostaly povolení "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 7]
```diff
@@ -3083,6 +3193,9 @@ msgid ""
 "returned to Amoran whenever he wanted to drink a good beer with friends and "
 "to tell some strange facts that he thought Glauco might be interested in."
 msgstr ""
+"Kroket se vrátil ke svému životu v lesích spolu se svou „přítelkyní“. Do "
+"Amoranu se ale vracel kdykoli si chtěl s přáteli vypít dobré pivo a "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"Amoranu se ale vracel, kdykoli si chtěl s přáteli vypít dobré pivo a "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 8]
```diff
@@ -3083,6 +3193,9 @@ msgid ""
...
+"poklábosit o nějakých nezvyklostech, která by podle něj mohly Glauka zajímat."
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"poklábosit o nějakých nezvyklostech, které by podle něj mohly Glauka zajímat."
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 9]
```diff
@@ -3092,6 +3205,10 @@ msgid ""
 "equally good company, and Glauco, every time he saw him coming, was careful "
 "not to let him never miss one or the other."
 msgstr ""
+"Kroket se vrátil ke svému životu v lesích, ale bez jeho „přítelkyně“ mu bylo "
+"smutno. A tak se vracel do Amoranu, kdykoli měl chuť na dobré pivo a dobrou "
+"společnost. Glauko, kdykoli ho viděl přicházet, dával pozor, aby nikdy mu "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"společnost. Glauko pokaždé, když ho viděl přicházet, dával pozor, aby mu nikdy "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 10]
```diff
@@ -3103,6 +3220,11 @@ msgid ""
 "prohibition of killing wolves in the territories subject to him, if not for "
 "personal defense, under penalty of expulsion from the territory."
 msgstr ""
+"Fangy zůstala s Glaukem až do konce bitvy proti Gan-Tarkasovi, protože tak "
+"by Kroket udělal. Pak zmizela a už ji nikdo nikdy neviděl! Glauko, který se "
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"by to Kroket udělal. Pak zmizela a už ji nikdo nikdy neviděl! Glauko, který se "
```

File: translations/cs/LC_MESSAGES/wesnoth-ala.po [hunk 11]
```diff
@@ -3103,6 +3220,11 @@ msgid ""
...
+"právě stal civilním guvernérem všech území na východě, na památku starého "
+"lovce nařídil pod trestem vyhnání z území absolutní zákaz zabíjení vlků na "
+"územích, která mu podléhala, s vyjímkou vlastní obrany."
```
Comment (celerini) [inline translations/cs/LC_MESSAGES/wesnoth-ala.po]:
```suggestion
"územích, která mu podléhala, s výjimkou vlastní obrany."
```

**Review by celerini — CHANGES_REQUESTED** [review]

**Comment by MultiSeptim** [comment]
Ready to merge.

**Pull request merged**